smithy-jsonschema/src/main/java/software/amazon/smithy/jsonschema/Schema.java

```diff
@@ -970,5 +1000,138 @@ public Builder disableProperty(String propertyName) {
                     return this;
             }
         }
+
+        Builder applyNode(String key, Node node) {
```

Comment:
This is a lot - can NodeMapper handle it?

Reply:
No, there's no way of adding custom behavior with unknown fields -`setWhenMissingSetter` takes an enum, `WhenMissing`. I guess we could add overload it, but adding a `fromNode` is nice and simple.